python/pyspark/sql/catalog.py

```diff
@@ -232,6 +232,11 @@ def clearCache(self):
         """Removes all cached tables from the in-memory cache."""
         self._jcatalog.clearCache()
 
+    @since(2.0)
+    def refreshTable(self, tableName):
+        """Invalidate and refresh all the cached the metadata of the given table."""
```

Comment:
There's an extra "the" in this comment, should be `refresh all the cached metadata`